floss/main.py

```diff
@@ -42,6 +43,16 @@
 
 MIN_STRING_LENGTH_DEFAULT = 4
 
+WINDOWS_FILES = ['KERNEL32.DLL', 'GDI32.DLL', 'USER32.DLL', 'COMCTL32.DLL',
+                 'COMDLG32.DLL', 'WS2_32.DLL', 'ADVAPI32.DLL', 'NETAPI32.DLL',
+                 'OLE32.DLL', 'SHELL32.DLL', 'WUSER32.DLL', 'PSAPI.DLL',
+                 'MSVCRT.DLL', 'CRTDLL.DLL', 'MSCOREE.DLL']
+
+IS_IP = "^([12]{,1}\d{1,2}\.){3}[12]{,1}\d{1,2}"
+IS_DOMAIN_FILE = "^[\\\\\w\/:@\.-]{5,256}\.[a-zA-Z0-9]{1,4}"
```

Comment:
can you add comments that describe:

  1. how these work?
  2. examples of what they should match?

the IP regex is neat - it took me a minute or two to figure out how it works.